### DISCUSSION TITLE
Does the connector support the online status check of certificates issued (e.g. Online Certificate Status Protocol, OCSP)?

### DISCUSSION THREAD

**Discussion by tdaskalopoulouSQS** [International-Data-Spaces-Association/DataspaceConnector#115] [General]
I have this dsc requirement.  **“Connector supports online status check of certificates issued (e.g. Online Certificate Status Protocol, OCSP)”**  I am not sure if there is an automatic verification to determine if the certificate is valid. If so, how can I check it?

**Comment by jmartinezsqs** [comment]
Hello

I comment on this issue as we need a solution to properly validate the functionality and safety of the connector.

We want to ensure whether the connector has the capability to validate a certificate, and where can we see if this is being performed.  This functionality is initially implemented as shown in the documentation with the criteria:

x     IAM 03     Online certificate status check

Let us know how to test if this criterion is properly working.

**Comment by SebastianOpriel** [comment]
Dear @jmartinezsqs 
Which certificate do you mean? I assume you mean DSC's IDS certificate e.g. for DAPS authentication, right?
Since version 6.4.0 actuator/info endpoint shows at least a certificate's expiration date (admittedly, not online status).
As DAPS is issuing DATs for valid certificates and DSC (if enabled) is validating DATs, DAPS could check the online status. I spoke with @bellebaum who opens an issue related to that feature (https://github.com/Fraunhofer-AISEC/omejdn-server/issues/48)
As IDS components also allow self-signed certificates, I would assume that no revocation list would be available in these situations.
